android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java

```diff
@@ -603,6 +605,64 @@ static void getPreviewImageAtPosition(String source, double sec, String format,
     promise.resolve(event);
   }
 
+  static void getTrimmerPreviewImages(String source, double startTime, double endTime, int step, String format, final Promise promise, ReactApplicationContext ctx) {
```

Comment:
Actually this method is not related to the androidx PR, anyway, we'll merge it

Reply:
not, those aren't related. Just bundled everything together for the PR, sorry